src/DssSpell.sol

```diff
@@ -21,14 +21,8 @@ pragma solidity 0.6.12;
 import "dss-exec-lib/DssExec.sol";
 import "dss-exec-lib/DssAction.sol";
 
-interface RwaLiquidationLike {
-    function ilks(bytes32) external view returns (string memory, address, uint48, uint48);
-    function init(bytes32, uint256, string calldata, uint48) external;
-    function bump(bytes32 ilk, uint256 val) external;
-}
-
-interface ChainlogLike {
-    function removeAddress(bytes32) external;
+interface OracleLike {
```

Comment:
it seems colliding with the one in `DssExecLib`, consider a different naming.

Reply:
Heads up this isn't done yet.